packages/fast-components-styles-msft/src/select/index.ts

```diff
@@ -67,7 +68,7 @@ const styles: ComponentStyles<SelectClassNameContract, DesignSystem> = {
         ...applyFloatingCornerRadius(),
         ...applyElevation(ElevationMultiplier.e11),
         zIndex: "1",
-        position: "absolute",
+        position: "relative",
         width: "100%",
         margin: "0",
         padding: format("{0} 0", toPx<DesignSystem>(designUnit)),
```

Comment:
Can we get rid of min/max width now? It seems like we don't need them, and there is no easy way to override them without styling the component.